marconi-chain-index/test/Spec/Marconi/ChainIndex/Indexers/Utxo/UtxoIndex.hs

```diff
@@ -174,7 +174,7 @@ propAllQueryUtxosShouldBeUnspent = Hedgehog.property $ do
         List.nub
           . fmap
             ( Utxo.QueryUtxoByAddressWrapper
-                . flip Utxo.QueryUtxoByAddress (Utxo.LessThanOrEqual upperBound) --  maxBound will overflow. See TODO below
+                . flip Utxo.QueryUtxoByAddress (Utxo.Interval Nothing (Just upperBound)) --  maxBound will overflow. See TODO below
```

Comment:
Would it worth creating a `lessthanOrEqual = Interval Nothing . Just` alias to ease creation with `Nothing` on the lower bound?

Reply:
Good idea! See https://github.com/input-output-hk/marconi/pull/110/commits/906f126982215a32d220b6ddd38713b671f12718